tests/lean/string_gaps.lean

```diff
@@ -0,0 +1,76 @@
+import Lean
```

Comment:
Please minimize the import and add docstrings https://lean-lang.org/lean4/doc/dev/testing.html#writing-good-tests